src/components/plan/PlanMergeReview.svelte

```diff
@@ -3,9 +3,9 @@
 <script lang="ts">
   import { goto } from '$app/navigation';
   import { base } from '$app/paths';
-  import BranchIcon from '@nasa-jpl/stellar/icons/branch.svg?component';
+  import MergeIcon from '@nasa-jpl/stellar/icons/merge.svg?component';
   import CheckIcon from '@nasa-jpl/stellar/icons/check.svg?component';
```

Comment:
I think CheckIcon should be above MergeIcon. Are your sort settings still disabled? That issue should be fixed in the latest VS Code.

Reply:
Aha, i did disable it and will update VS code, thanks!

Reply:
Done